### DISCUSSION TITLE
Where is the Motif Updated Dock Score (MUDS) value after a Replica Dock 2.0 global run?

### DISCUSSION THREAD

**Discussion by ww2283** [RosettaCommons/rosetta#219] [Rosetta]
Hello, I did a Replica Dock 2.0 global run, where can I find the Motif Updated Dock Score (MUDS) value? I'd like to plot the value against the RMSD for a funnel plot. Thanks

**Comment by roccomoretti** [comment]
I believe the actual weights file for MUDS is located at `database/scoring/weights/muds_2021.wts` This is referenced in the provided replicadock.xml script as `score_dock_low`. From my read of the given XML, there isn't any deliberate explicit score output of the `score_dock_low`/muds score. However, you can cause RosettaScripts to explicitly rescore the structure with that scorefunction by adding `<OUTPUT scorefxn="score_dock_low" />` between the `</PROTOCOLS>` and `</ROSETTASCRIPTS>` line. This will result in the `total_score` line in the output PDBs and scorefile to be the values for the muds_2021.wts scorefunction.

If you already have structures, you should be able to re-score them with this scorefunction by using the `score_jd2` application and the `-score:weights muds_2021` option. (Again, the total_score column of the output scorefile will be the MUDS score.) You'll likely also have to include all the other options which define the motif parameters in order to get accurate scoring.

**Comment by ww2283** [comment]
Thank you for your prompt response. I have a subsequent question regarding the scoring part. In my current scenario, while utilizing Replica Dock 2.0 for docking, I lack an actual native structure. Consequently, I used a reasonably predicted structure as the native reference. After calculating the MUDS, is it acceptable to select a structure with a superior score as the 'native' and proceed with funnel-like plotting? Alternatively, should I consider rerunning the docking using this newly selected 'native' structure?

**Comment by roccomoretti** [comment]
Yes, when doing score-vs-rmsd plots in actual experimental systems (where the native isn't available) it's typical to pick a representative output structure as the reference structure for the rmsd calculation. The concept is that if the protocol is working and the reference structure is sufficiently native-like, then your score-vs-rmsd plot to your assumed-native-like reference structure will be similar to the score-vs-rmsd plots produced during benchmarking using actual natives. 

If they're not, then either the protocol isn't working properly, or the assumption that the chosen reference structure is native-like was mistaken. Often times you'll pick several different prospective native structures and plot the score-vs-rmsd plot for each of them. You may find that one (not necessarily the lowest energy one) gives a relatively decent "native like" funnel, which provides support for the claim that that reference structure is a native-like one.
